tests/test_uploads.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from unittest.mock import patch
+
+from jwplatform import __version__
+from jwplatform.client import JWPlatformClient
+
+# from .mock import JWPlatformMock
+
+
+def test_upload(api_secret):
```

Comment:
We'll probably want some unit tests to replace this.

Reply:
Unit tests have been added.